### DISCUSSION TITLE
publish stream from Smartphone (flutter-app) to Server, and forward udp-Multicast

### DISCUSSION THREAD

**Discussion by sandmen** [bluenviron/mediamtx#1400] [General]
Hi, 
I have read the documentation, but maybe I have overseen some thing.
My use case, smartphone should publish a stream to a server, where rtsp-simple-server is running.
Smartphone has LTE, and server is accessible from internet.
Now, i want to forward this stream to several devices (smartphone,Laptop....) I think, 10-20 users.

Is that a use case for rtsp-simple server ?
Any example configs for that ?

Thanks, for your feedback.

**Comment by aler9** [comment]
Hello, this is supported with the standard configuration. You may need to disable the UDP protocol, which is not fit for the internet:

```yml
protocols: [tcp]
```

You can then publish to

```
rtsp://server-ip:8554/stream
```

and read the stream with RTSP, RTMP, HLS or WebRTC from any device.

**Comment by francisuk1989** [comment]
Keep in mind that 4G/LTE some provider will only allow ports 80/443 traffic outgoing in some country's and incoming ports will be likely blocked under CGNAT however there is few Mobile ISPs that will give you a Public IP facing for incoming connections.

RTMP Camera for Android.
https://play.google.com/store/apps/details?id=rtmp.ip.security.camera

and then with this app you can do what @aler9 has suggested above.

> Hello, this is supported with the standard configuration. You may need to disable the UDP protocol, which is not fit for the internet:
> 
> ```yaml
> protocols: [tcp]
> ```
> 
> You can then publish to
> 
> ```
> rtsp://server-ip:8554/stream
> ```
> 
> and read the stream with RTSP, RTMP, HLS or WebRTC from any device.

[a comment by an automated account is not shown]
